index.json

```diff
@@ -106,6 +117,6 @@
     "sm": "2px",
     "md": "4px",
     "lg": "8px",
-    "pill": "999px"
+    "pill": "50%"
```

Comment:
Doesn't this render kind of wonky if you use percentages? I think we're using 999px because percentages have weird scaling:

![yYNPa8O4](https://user-images.githubusercontent.com/705973/96923437-4bcc1480-147f-11eb-90da-b4f15937ed63.gif)

Reply:
What a strange CSS bug. TIL 👍 

I'll switch it back. I already created a global mixin for the same result: https://github.com/iFixit/core-design/blob/master/packages/components-web/src/assets/style/global.less#L19